Bottom: c1ab11d740a32967db76710c9719b730964ab841
Top:    739a68b4f57f0f0eb2ed2dff067ebc5d7d38c39d
Author: Sean Hefty <sean.hefty@intel.com>
Date:   2013-08-02 14:17:03 -0700

Refresh of rs-setroute

---

diff --git a/src/rsocket.c b/src/rsocket.c
index 7768f58..e3406a4 100644
--- a/src/rsocket.c
+++ b/src/rsocket.c
@@ -3326,6 +3326,7 @@ int rsetsockopt(int socket, int level, int optname,
 			if ((rs->optval = calloc(optlen, 1))) {
 				memcpy(rs->optval, optval, optlen);
 				rs->optlen = optlen;
+				ret = 0;
 			} else {
 				ret = ERR(ENOMEM);
 			}
